haproxy/config.sls

```diff
@@ -1,13 +1,16 @@
+{% from "haproxy/map.jinja" import haproxy with context %}
+
+{% set config_file = salt['pillar.get']('haproxy:config_file_path', haproxy.config_file) %}
 haproxy.config:
  file.managed:
-   - name: {{ salt['pillar.get']('haproxy:config_file_path', '/etc/haproxy/haproxy.cfg') }}
-   - source: salt://haproxy/templates/haproxy.jinja
+   - name: {{ config_file }}
+   - source: {{ haproxy.config_file_source }}
```

Comment:
This changes the pillar key from `haproxy:config_file_path` to `haproxy:config_file_source`

Reply:
No, it doesn't. In map.jinja config_file is the destination file on the server and config_file_source is the source on Salt master.
That means the 'ideal' pillar to override config file is 'haproxy: lookup: config_file', but to kept actual behavious there is this line https://github.com/saltstack-formulas/haproxy-formula/pull/69/files#diff-44c85aa6e52c67554f24e08baf667cfbR3 which get pillar from 'haproxy: config_file_path'